src/sb_pointcloud_processing/src/test_pcl_generator_node.cpp

```diff
@@ -100,25 +108,29 @@ sensor_msgs::PointCloud2 generatePclMessage(bool include_outlier) {
     // Generate a single PointCloud with noise
     pcl::PointCloud<pcl::PointXYZ> pcl;
     LineExtractor::TestUtils::addLineToPointCloud(
-    args, pcl, max_noise_x, max_noise_y);
+    args, pcl, max_noise_x, max_noise_y, getSeed());
 
     // Add second line to the pointcloud
     args.coefficients = second_line;
     LineExtractor::TestUtils::addLineToPointCloud(
-    args, pcl, max_noise_x, max_noise_y);
+    args, pcl, max_noise_x, max_noise_y, getSeed());
 
     // Add outlier if wanted
     if (include_outlier) {
         args.coefficients = outlier_line;
         args.x_delta      = outlier_x_delta;
 
         LineExtractor::TestUtils::addLineToPointCloud(
-        args, pcl, max_noise_x, max_noise_y);
+        args, pcl, max_noise_x, max_noise_y, getSeed());
     }
 
     // convert pointcloud to sensor msgs
     sensor_msgs::PointCloud2 msg;
     pcl::toROSMsg(pcl, msg);
 
     return msg;
-}
\ No newline at end of file
+}
+
+unsigned int getSeed() {
+    return counter++ % 10;
```

Comment:
We should add a javadoc comment for this function. Also, why are we `% 10`'ing this? won't this mean we will only ever generate 10 seeds?

Reply:
Yes, the intention was to visually (because it's rviz) ensure that the test works for 10 seeds. If we have `MAX_INT` number of seeds, we can't guarantee that the test works for all of them unless we wait/watch long enough